run_ci.sh

```diff
@@ -140,6 +140,9 @@ for PROJ in installer ; do
     mv /home/notstack/$PROJ $HOME/go/src/github.com/$GITHUB_ORGANIZATION
 done
 
+# If you wish the deployment to install a custom MAO then set the following env variable
+export TEST_CUSTOM_MAO=true
```

Comment:
Can you document this in config_example.sh / README.md ?

how about variables for the repo and branch, as well?

Reply:
Yeah, the image repo and branch can be passed in via CLI set and env variables or use defaults. Will update documentation.